src/components/TodoList/TodoList.tsx

```diff
@@ -1,210 +1,115 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useMemo } from 'react';
+import cn from 'classnames';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions as currentTodoActions } from '../../features/currentTodo';
+
+interface Props {
+  isLoading: boolean
+}
+
+export const TodoList: React.FC<Props> = ({ isLoading }) => {
+  const todos = useAppSelector(state => state.todos);
+  const { status, query } = useAppSelector(state => state.filter);
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const dispatch = useAppDispatch();
+
+  const visibleTodos = useMemo(() => {
+    let filteredTodos = todos;
+
+    switch (status) {
+      case 'completed':
+        filteredTodos = filteredTodos.filter(todo => todo.completed);
```

Comment:
lets put this into separate variable, we use this logic twice so let's not repeat same code
`filteredTodos = filteredTodos.filter(todo => todo.completed);`